doc/source/user_guide/tutorials/animate/animate_data.rst

```diff
@@ -0,0 +1,503 @@
+.. _ref_tutorials_animate_data:
+
+============
+Animate data
+============
+
+:bdg-warning:`MAPDL` :bdg-success:`LSDYNA`
+
+.. |Field| replace:: :class:`Field<ansys.dpf.core.field.Field>`
+.. |FieldsContainer| replace:: :class:`FieldsContainer<ansys.dpf.core.fields_container.FieldsContainer>`
+.. |Fields| replace:: :class:`Fields<ansys.dpf.core.field.Field>`
+.. |FieldsContainers| replace:: :class:`FieldsContainers<ansys.dpf.core.fields_container.FieldsContainer>`
+.. |MeshedRegion| replace:: :class:`MeshedRegion <ansys.dpf.core.meshed_region.MeshedRegion>`
+.. |TimeFreqSupport| replace:: :class:`TimeFreqSupport <ansys.dpf.core.time_freq_support.TimeFreqSupport>`
+.. |animate| replace:: :func:`animate() <ansys.dpf.core.fields_container.FieldsContainer.animate>`
+.. |Result| replace:: :class:`Result <ansys.dpf.core.results.Result>`
+.. |Operator| replace:: :class:`Operator<ansys.dpf.core.dpf_operator.Operator>`
+
+This tutorial shows how to animate your results data.
+
+To animate a result you need to get the data stored in a |FieldsContainer| labeled in time steps.
+
+Get the result files
+--------------------
+
+Here we we will download a  result file available in our `Examples` package.
```

Comment:
```suggestion
First download a result file available in our `Examples` package.
```

Reply:
We may need a `replace` for the `ansys.dpf.core.examples` module like you did for `FieldsContainer`.